lib/msal-node/test/client/ConfidentialClientApplication.spec.ts

```diff
@@ -203,7 +203,7 @@ describe("ConfidentialClientApplication", () => {
             auth: {
                 clientId: TEST_CONSTANTS.CLIENT_ID,
                 authority: TEST_CONSTANTS.AUTHORITY,
-                clientAssertion: "testAssertion",
+                clientAssertion: () => "testAssertion",
```

Comment:
Why did this have to change? Returning a string should still work?

Reply:
I'll be updating the unit tests shortly. I wanted to see if you approved of the proof-of-concept in this PR.